assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -0,0 +1,45 @@
+const { __ } = wp.i18n;
+
+export const CampaignCovers = ({ covers, covers_view, row, loadMoreCovers }) => {
+  const rowAmount = 3;
+  const showLoadMore = (covers.length > rowAmount && covers_view === '1') ||
+    (covers.length > (rowAmount * 2) && covers_view === '2');
```

Comment:
We should probably replace these numbers with an actual descriptive name, I don't see why it would need to be a number. It's a bit more work to do it now, since we would need to write a function for the `deprecated` entry to map the old numbers to the new string, but it seems worth it, since it will make the block comment a lot easier to understand.

Reply:
Maybe a first step would be to replace all these numbers with constants that have the number as a value, so we can at least understand the code :sweat_smile:

Reply:
These actually kind of make sense since they are the amount of rows: either 1, 2 or all. So the only possible value that doesn't really make sense is '3', which stands for 'all rows'... But I can do something similar as what I did for the `cover_type` attribute which was also a mysterious number and now has constants?

Reply:
Right, I first thought it contained a type, since it's a string and not an int. Maybe better to make it an int and use it directly? We could call it `initialRowsLimit`, and wouldn't need a special value for the "no limit" case, we could instead leave it empty in that case.

Reply:
Then the equation would be much simpler:
```
const showLoadMore = !!initialRowsLimit && covers.length > (amountPerRow * initialRowsLimit)

Reply:
Updated! 🙂